tests/app/delivery/test_send_to_providers.py

```diff
@@ -736,48 +736,59 @@ def test_send_email_to_provider_uses_reply_to_from_notification(
 def test_send_sms_to_provider_should_return_template_if_found_in_redis(
         mocker, client, sample_template
 ):
-
-    from app.schemas import template_schema
+    from app.schemas import service_schema, template_schema
```

Comment:
This line is repeated in the the test for emails. Does it make sense to DRY it up as a global import for the module?

Reply:
The service_schema and template_schema need to be imported for the test locally because of the dependency conflicts with Marshmallow and SQLAlchemy. 
We get ` AttributeError: 'DummySession' object has no attribute 'query'` error if you globally import the schemas.

Reply:
Ah, OK. Thanks for explaining. Sounds like an annoying quirk! It would be helpful to add that to the commit message, as someone in future may also ask why the import is at the function level.